### DISCUSSION TITLE
Support shared libraries

### DISCUSSION THREAD

**Discussion by tiztrinity** [BartmanAbyss/vscode-amiga-debug#254] [Q&A]
Hi,
is it possible to create my own shared .library file using the current toolchain?
What I would like to achieve is to put some code inside a library and load/unload it on demand using OpenLibrary/CloseLibrary

Thanks!

**Comment by BartmanAbyss** [comment]
I think so. afaik .library files are also just hunk exe files.
Maybe have a look at this project and adapt the makefile?
https://github.com/alexalkis/library

However, be warned that you will not be able to debug (set breakpoints, watch variables) in .library files loaded by OpenLibrary
